docs/Contact.md

```diff
@@ -0,0 +1,18 @@
+# Contacting the HLSL Team
+
+This page lists some of the public spaces where you can find members of the HLSL.
+
+## Discord Channels
+
+* [DirectX](https://discord.com/invite/directx)
+* [LLVM](https://discord.gg/xS7Z362)
+
+## GitHub Issues
+
+* [DirectXShaderCompiler](https://github.com/microsoft/DirectXShaderCompiler/issues/new)
+* [hlsl-specs](https://github.com/microsoft/hlsl-specs/issues/new)
+* [LLVM](https://github.com/llvm/llvm-project/issues/new)
+
+## Email
+
+* [Ask HLSL](mailto:askhlsl@microsoft.com)
```

Comment:
It might be useful to suggest what each resource is most appropriate for.  My perfunctory suggestions:

DirectX Discord - D3D related HLSL topics
LLVM discord - Modernization project HLSL and other DX topics

DXC Github - bugs and feature requests specific to the DXC compiler
LLVM Github - bugs and feature requests specific to the modern clang HLSL/DX compiler
HLSL-Specs - feature requests for language and shader model versions

ask HLSL email - when privacy is essential or to prod a submission in any of the other contact methods.